src/app/users/users.component.scss

```diff
@@ -0,0 +1,25 @@
+h2 {
+  text-align: center;
+}
+table {
+  width: 100%;
+  margin: 0 auto;
+}
+
+table, tr, td {
+  border: 1px solid;
+  border-spacing: 0;
```

Comment:
default

Reply:
border nebo border-spacing ?
u me ve chromu to neni default :D takze nevim :D